generators/app/index.js

```diff
@@ -90,14 +90,14 @@ module.exports = generators.Base.extend({
             return false;
           }
 
-          var done = this.async();
+          var cb = this.async();
```

Comment:
Why did you rename this? Please keep unrelated changes out of this commit :wink: